models/truckModel.js

```diff
@@ -3,13 +3,18 @@
 // name your model 'Truck' (see the example in the slide deck for 'Book')
 // this is to match the name of the collection the data was imported into
 
-// var foodTruckSchema = new Schema({
-// 	name: String,
-// 	foodType: [String],
-// 	schedule: [String],
-// 	payment: [String],
-// 	description: String,
-// 	website: String,
-// 	Facebook: String,
-// 	Twitter: String
-// });
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+ var foodTruckSchema = new Schema({
```

Comment:
your Schema should be the Schema already provided - the type parameters would work, but you still have to indicate the foodType, schedule, and payment values as arrays of Strings as indicated in the Schema already provided